### ISSUE TITLE
Easier entry of address by hand

### ISSUE THREAD

**Issue by ThePiGuy** [spesmilo/electrum#1363]
When you're entering an address by hand (which I have to do occasionally), it would be great if the address entry field turned red or gave some kind of visual (but not distracting [like a popup box]) cue to tell me that I made a mistake.  Ideally this would be best if it did the check after every character typed, and then gave you a typo warning upon starting to type the checksum.

What would make it way more awesome (and I don't know if this is totally possible or could be implemented effectively), but if, once you finish typing the checksum, it started doing permutations of the address (in the background though, don't show the user the tests, just highlight the character in red if a possible solution is found), changing one character at a time to its capital form (and checking if it then matches the checksum), then to its lowercase form, then to a few surrounding characters on the keyboard.

TLDR: turn the address input field red as soon as an error is detected in the entry of an address.

**Comment by ecdsa** [comment]
who enters addresses by hand?

**Comment by ThePiGuy** [comment]
Goofballs like myself.  Not that it's a huge market, but it would tell you if you miscopied something off a website or anything.

**Comment by ecdsa** [comment]
wontfix, sorry. The address input field needs to be compatible with aliases (OpenAlias), which makes your suggestion too complicated.

**Comment by ThePiGuy** [comment]
How about, instead of saying "Invalid lines", it displays a little red X beside the field when you click Send to show you that it's invalid.
